tests/subworkflows/nf-core/align_bowtie2/test.yml

```diff
@@ -0,0 +1,30 @@
+- name: align bowtie2
+  command: nextflow run ./tests/subworkflows/nf-core/align_bowtie2 -entry test_align_bowtie2 -c tests/config/nextflow.config
+  tags:
+    - subworkflows/align_bowtie2
+    - subworkflows/align_bowtie2
+    - subworkflows/bam_sort_samtools
+    - subworkflows/bam_stats_samtools
+    # Modules
+    - bowtie2
+    - bowtie2/align
+    - samtools
+    - samtools/index
+    - samtools/sort
+    - samtools/stats
+    - samtools/idxstats
+    - samtools/flagstat
```

Comment:
Do I get this right that this should trigger the subworkflow test every time one of those modules is changed?

Reply:
Exactly! Also, if you test locally and use the tag instead of the test name, then you'll run any subworkflows that depend on it. So doing it this way we get the added benefit of being locked into pytest-workflow, but not locked in to GitHub and the check for changes action(I'm more afraid they'll change something underneath us).

Reply:
Amazing 💥